spring/spring-web/src/main/java/io/quarkus/ts/spring/web/boostrap/web/SimpleController.java

```diff
@@ -19,7 +20,8 @@
     @Inject
     Template home;
 
-    @RequestMapping(value = "/", produces = MediaType.TEXT_HTML)
+    // TODO https://github.com/quarkusio/quarkus/issues/20278
+    @RequestMapping(value = "/", produces = MediaType.TEXT_HTML, method = RequestMethod.GET)
```

Comment:
present in Quarkus 2.2.3 or just in Quarkus main?

Reply:
Quarkus 2.2.3.Final (The following combination works): 
`quarkus-spring-web`
`quarkus-resteasy-qute`

Quarkus 2.2.3.Final (doesn't work): 
`quarkus-spring-web`
`quarkus-resteasy-reactive-qute`

Quarkus upstream is the opposite
Works:
`quarkus-spring-web`
`quarkus-resteasy-reactive-qute`

Doesn't works
`quarkus-spring-web`
`quarkus-resteasy-qute`